build.toml

```diff
@@ -117,13 +117,25 @@ target = 'Linux RT x64'
 build_spec = 'Engine Release'
 dependency_target = 'Linux_x64'
 
-[package]
+[[package]]
 type = 'nipkg'
 control_file = 'control_dsf_cd'
 package_output_dir = 'Built'
 
 [package.payload_map]
 'Built\\Custom Device' = 'documents\National Instruments\NI VeriStand {veristand_version}\Custom Devices\Data Sharing Framework'
-'Built\\Scripting API' = 'ni-paths-LV{veristand_version}DIR\vi.lib\addons\VeriStand Custom Device Scripting APIs\Data Sharing Framework'
-'Built\\Scripting Examples' = 'ni-paths-LV{veristand_version}DIR\examples\NI VeriStand Custom Devices\Data Sharing Framework'
 'Built\\Errors' = 'ni-paths-NISHAREDDIR\Errors\English'
```

Comment:
My gut feel is that the custom device should be installing an errors file to the year-version specific runtime directory `ni-paths-NISHAREDDIR\LabVIEW Run-Time\{veristand_version}\errors\English`, and that the scripting API should be installing it to `ni-paths-NISHAREDDIR\Errors\English`.

@buckd thoughts?

Reply:
we currently have two distinct error files: runtime errors and scripting errors

Reply:
How do these two distinct files get installed? Presumably to distinct locations?

Reply:
in this repo:
'<Build>\<LV version>\Errors\DSF-errors.txt' mapped to 'ni-paths-NISHAREDDIR\Errors\English' 

in the plugins repo:
<Build>\<LV Version>\Errors\DSF-GE-ReflectiveMemory-errors.txt' mapped to 'ni-paths-NISHAREDDIR\LabVIEW Run-Time\{veristand_version}\errors\English'

on the plugins error file, we have previously noted the issue with installation of the (same) error file to a version-agnostic folder through different, version-specific packages .

Reply:
I see - are there error codes that the custom device itself generates, which don't come from the plugins?

Reply:
I'm fine moving this discussion off of this PR - it's not worth gating this obvious improvement on a mostly philosophical discussion on how we should install error files.

I'll follow up asynchronously.

Reply:
I think Core returns LV errors. As we widen the yellow-brick road, we will likely create DSF-specific error codes with more context-specific description.